src/lib/symlink-manager.ts

```diff
@@ -150,11 +152,7 @@ export class SymlinkManager {
 			return;
 		}
 
-		if ( ! this.php.requestHandler ) {
-			throw new Error( 'Request handler is not set' );
-		}
-
-		const vfsPath = path.posix.join( this.php.requestHandler.documentRoot, filename );
+		const vfsPath = path.posix.join( this.documentRoot, filename );
```

Comment:
WP-cli doesn't use request handlers, so we need to enable the Symlink manager to work without assuming that a request handler will be available.